src/net_processing.h

```diff
@@ -73,6 +73,14 @@ class PeerLogicValidation final : public CValidationInterface, public NetEventsI
     */
     bool SendMessages(CNode* pto) override EXCLUSIVE_LOCKS_REQUIRED(pto->cs_sendProcessing);
 
+    /**
+    * Process global tasks that aren't attached to a specific peer
+    *
+    * @param[in]   interrupt       Interrupt condition for processing threads
+    * @returns     bool            true if there's more work to do
+    */
+    bool ProcessGlobalTasks(std::atomic<bool>& interrupt);
```

Comment:
Compiler warning `-Winconsistent-missing-override`
```suggestion
    bool ProcessGlobalTasks(std::atomic<bool>& interrupt) override;
```

Reply:
Good catch. Fixed!